src/file.js

```diff
@@ -12,6 +12,59 @@ var utils = require('./utils');
 
 var file = module.exports;
 
+file.copyFile = function(src, dst, opts, cb) {
+	/* Wrapper for copyFile function with code to handle it not being present. If/When nodejs 8.5 or greater is a baseline requirement/assumption,
+	   this can be pruned and wholesale replace in all calls with fs.copyFile. */
+
+	function noop() {}
+
+  /* Test for availability of fs.CopyFile */
+	const semver = require('semver');
```

Comment:
Why not just use `typeof fs.copyFile !== 'function'`.

Also, you need to fix the formatting below. It doesn't match our linter at all.